src/Decoder/SmileDecoder.php

```diff
@@ -290,7 +277,7 @@ private function writeFloat(int $byte): string
         };
     }
 
-    private function writeAsciiOrUnicode(int $byte, int $bits, bool $isKey = false): string
+    private function writeUnicodeOrASCII(int $byte, int $bits, bool $isKey = false): string
```

Comment:
```suggestion
    private function writeUnicodeOrAscii(int $byte, int $bits, bool $isKey = false): string
```

In PHP symbol, otherwise, we use camel case.